src/katello/tracer.py

```diff
@@ -4,7 +4,7 @@
 import imp
 
 def collect_apps():
-    raise Exception("This is not the method you want to use. Normally, this method should be provided by apt_tracer or zypper_tracer")
+    raise Exception("This is not the method you want to use. Normally, this method should be provided by deb_tracer or zypper_tracer")
```

Comment:
Unrelated, but I'd expect it to use `NotImplementedError` since it's more specific:
```suggestion
    raise NotImplementedError("This is not the method you want to use. Normally, this method should be provided by deb_tracer or zypper_tracer")
```